CONTRIBUTING.md

```diff
@@ -61,7 +106,23 @@ A couple notes:
      puppet/puppet-agent-ubuntu apply --modulepath=$PWD examples/init.pp
    ```
 
-   Note that if you want to run manifests directly then need to be mounted into the container.
+Note that if you want to run manifests directly then need to be mounted into the container.
+
+## Testing
+
+See [jenkins.sh](jenkins.sh).
```

Comment:
Oof

Reply:
I know. this should be improved, but out of scope rn.